src/platforms/salesforce/actions/mappers.ts

```diff
@@ -1,8 +1,17 @@
 import {
   SalesforceContact,
   SalesforceListView,
+  SalesforceQueryRecord,
   SalesforceUser,
 } from '@/platforms/salesforce/schemas';
+import { camel, mapKeys, omit } from 'radash';
+
+export const transformRecord = (record: SalesforceQueryRecord) => {
+  return {
+    id: record.Id,
+    ...mapKeys(omit(record, ['$native']), camel),
```

Comment:
did we decide to camel case for the client?

Reply:
Yeah, we enforce it here
https://github.com/vesselapi/integrations/blob/37023b5c1b66dc62b2fd942092f3ba223771efbe/src/sdk/types.ts#L236